tests/cases/standalone/common/aggregate/distinct_order_by.sql

```diff
@@ -4,6 +4,14 @@ INSERT INTO integers VALUES (1), (2), (3);
 
 SELECT DISTINCT i%2 FROM integers ORDER BY 1;
 
+-- TODO(LFC): Failed to run under new DataFusion
+-- expected:
+--  +-----------------------+
+--  | integers.i % Int64(2) |
+--  +-----------------------+
+--  | 1                     |
+--  | 0                     |
+--  +-----------------------+
 SELECT DISTINCT i % 2 FROM integers WHERE i<3 ORDER BY i;
```

Comment:
Just comment out this case instead of changing its expected output?
```suggestion
-- SELECT DISTINCT i % 2 FROM integers WHERE i<3 ORDER BY i;
```

Reply:
I think it's better to keep it. Were DataFusion fixed the problem, we could easily figure out -- the sqlness tests "failed".